config/jobs/kubernetes-sigs/cluster-api-provider-azure/cluster-api-provider-azure-periodics.yaml

```diff
@@ -71,3 +72,39 @@ periodics:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-azure
     testgrid-tab-name: periodic-capi-e2e
     testgrid-alert-email: k8s-infra-staging-cluster-api-azure@kubernetes.io
+- name: periodic-cluster-api-provider-azure-coverage
+  interval: 12h
+  decorate: true
+  path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+  extra_refs:
+    - org: kubernetes-sigs
+      repo: cluster-api-provider-azure
+      base_ref: master
+      path_alias: "sigs.k8s.io/cluster-api-provider-azure"
+    - org: kubernetes
+      repo: test-infra
+      base_ref: master
+      path_alias: k8s.io/test-infra
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/kubekins-e2e:v20200819-491a5ae-1.18
+      command:
+      - runner.sh
+      - bash
+      args:
+      - -c
+      - |
+        result=0
+        ./scripts/ci-test-coverage.sh || result=$?
+        cp coverage.* ${ARTIFACTS}
+        cd ../../k8s.io/test-infra/gopherage
+        GO111MODULE=on go build .
+        find ${ARTIFACTS} -type f -iname 'junit*.xml' -exec rm {} \;
+        ./gopherage filter --exclude-path="zz_generated,generated\.pb\.go" "${ARTIFACTS}/coverage.out" > "${ARTIFACTS}/filtered.cov" || result=$?
+        ./gopherage html "${ARTIFACTS}/filtered.cov" > "${ARTIFACTS}/coverage.html" || result=$?
+        ./gopherage junit --threshold 0.05 "${ARTIFACTS}/filtered.cov" > "${ARTIFACTS}/junit_coverage.xml" || result=$?
```

Comment:
This threshold value will fail anything under 5%, I used it just as a marker for low/zero coverage areas. But if you have more strict req you can up it.